### ISSUE TITLE
Unable to preview PDF

### ISSUE THREAD

**Issue by nathanmerrill** [standardnotes/forum#3268]
**Describe the bug**
I've attached a PDF to one of my notes.  When I go to files, and click on it, it says "This file can only be previewed in an external app".  I click on the button "Open file preview", and nothing happens.  I have MuPDF viewer installed: https://play.google.com/store/apps/details?id=com.artifex.mupdf.viewer.app  which does work if I preview from the filesystem.

**To Reproduce**
1. Upload a pdf.  The pdf here does not matter as far as I can tell.
2. Open the file, either from the Files tab or from the note itself.
3. Click on "Open file preview".  Nothing happens.

I thought this might be related to my pdf viewer, but I've installed Foxit PDF viewer, and that didn't change anything either.

**Expected behavior**
I'd expect a file preview of the app to open

**Smartphone (please complete the following information):**
 - Device: Nord N200
 - OS: LineageOS version 20
 - Version: Android version 13

**Comment by effieeee** [comment]
Hi @nathanmerrill, I’m sorry to hear about the issues you’re experiencing, though I’m afraid I’m not able to test on LineageOS at the moment, so I’m still not certain if this might be an issue specific to Lineage. Are you also unable to preview the PDF when using the web app through any browser? (app.standardnotes.com)

**Comment by nathanmerrill** [comment]
Yes, I can preview the PDF in both my android browser as well as the browser on my computer.
